src/Raven.Studio/typescript/viewmodels/manage/threadStackTrace.ts

```diff
@@ -0,0 +1,72 @@
+import dialogViewModelBase = require("viewmodels/dialogViewModelBase");
+import copyToClipboard = require("common/copyToClipboard");
+import getStackTraceForThreadCommand = require("commands/maintenance/getStackTraceForThreadCommand");
+import generalUtils = require("common/generalUtils");
+
+class threadStackTrace extends dialogViewModelBase {
+
+    threadId = ko.observable<number>();
+    threadName = ko.observable<string>();
+    threadType = ko.observable<string>();
+    
+    cpuUsage = ko.observable<string>();
+    stackTrace = ko.observableArray<string>([]);
+    
+    dialogContainer: Element;
+
+    spinners = {
+        loading: ko.observable<boolean>(false)
+    };
+
+    constructor(threadId: number, threadName: string) {
+        super();
+        
+        this.threadId(threadId);
+        this.threadName(threadName);
+    }
+    
+    compositionComplete() {
+        super.compositionComplete();
+        this.dialogContainer = document.getElementById("threadStackTraceDialog");
+    }
+    
+    activate() {
+        return this.loadStackTrace(this.threadId());
+    }
+
+    private loadStackTrace(threadId: number): JQueryPromise<threadStackTraceResponseDto> {
+        this.spinners.loading(true);
+        this.stackTrace([]);
```

Comment:
I wonder if we need that.

Reply:
well, not black or white, but when you have this it gives the impression that the info is 'new'
otherwise, if info from ep (after refresh) is the **same** you kind of wonder

Reply:
let's remove that line

Reply:
done